sdk/cognitiveservices/cognitiveservices-qnamaker/README.md

```diff
@@ -24,35 +24,40 @@ npm install @azure/ms-rest-azure-js
 ```
 
 ##### Sample code
-The following sample lists the keys of the QnAMaker endpoint. To know more, refer to the [Azure Documentation on QnAMaker](https://docs.microsoft.com/en-us/azure/cognitive-services/qnamaker/)
 
 ```typescript
-import { QnAMakerClient } from "@azure/cognitiveservices-qnamaker";
 import { CognitiveServicesCredentials } from "@azure/ms-rest-azure-js";
+import { QnAMakerClient } from "@azure/cognitiveservices-qnamaker";
+
+async function main() {
+  const qnaMakerKey = process.env["qnaMakerKey"] || "<qnaMakerKey>";
```

Comment:
Generally environment variables are formatted in `UPPER_CASE` so I think the original env var names were more appropriate for this example.  Do you mind changing them back here and the code later in this file?  The rest of the sample changes look good.